rest/route/github.go

```diff
@@ -667,6 +683,36 @@ func (gh *githubHookApi) requireSigned(ctx context.Context, userRepo data.UserRe
 	return nil
 }
 
+func (gh *githubHookApi) requireApprovals(ctx context.Context, userRepo data.UserRepoInfo, prNum, requiredApprovalCount int) error {
+	settings, err := evergreen.GetConfig()
+	if err != nil {
+		return errors.Wrap(err, "getting admin settings")
+	}
+
+	githubToken, err := settings.GetGithubOauthToken()
+	if err != nil {
+		return errors.Wrap(err, "getting GitHub OAuth token from settings")
+	}
+
+	reviews, err := thirdparty.GetGithubPullRequestReviews(ctx, githubToken, userRepo.Owner, userRepo.Repo, prNum)
+	if err != nil {
+		return errors.Wrap(err, "getting GitHub commits")
+	}
+
+	var num_approvals int
+	for _, r := range reviews {
+		state := r.GetState()
+		if state == reviewApproved {
+			num_approvals += 1
+		}
+	}
+
+	if num_approvals < requiredApprovalCount {
+		return errors.Errorf("PR '%d' does not have enough approvals", prNum)
```

Comment:
Integers formatted with `%d` don't need surrounding quotes. The quotes are only needed for strings because they can be empty.

Reply:
fixed